gridy/Sources/Presentations/PlanBoard/TimelineLayout/ScheduleAreaView.swift

```diff
@@ -1,52 +1,136 @@
+                        let startY = CGFloat(line.startRow + 1) * viewStore.scheduleAreaGridHeight
+                        let startX = CGFloat(line.startCol) * viewStore.gridWidth - CGFloat(viewStore.shiftedCol) * viewStore.gridWidth
+                        let endX = CGFloat(line.endCol + 1) * viewStore.gridWidth - CGFloat(viewStore.shiftedCol) * viewStore.gridWidth
+                        
+                        Path { path in
+                            path.move(to: CGPoint(x: startX, y: startY))
+                            path.addLine(to: CGPoint(x: endX, y: startY))
+                        }
+                        .stroke(Color.black, lineWidth: 2)
+                        
+                        let diamondSize: CGFloat = 10
+                        let diamondStart = CGPoint(x: startX, y: startY)
+                        let diamondEnd = CGPoint(x: endX, y: startY)
+                        
+                        Path { path in
+                            path.move(to: CGPoint(x: diamondStart.x, y: diamondStart.y - diamondSize / 2))
+                            path.addLine(to: CGPoint(x: diamondStart.x + diamondSize / 2, y: diamondStart.y))
+                            path.addLine(to: CGPoint(x: diamondStart.x, y: diamondStart.y + diamondSize / 2))
+                            path.addLine(to: CGPoint(x: diamondStart.x - diamondSize / 2, y: diamondStart.y))
+                            path.closeSubpath()
+                            
+                            path.move(to: CGPoint(x: diamondEnd.x, y: diamondEnd.y - diamondSize / 2))
+                            path.addLine(to: CGPoint(x: diamondEnd.x + diamondSize / 2, y: diamondEnd.y))
+                            path.addLine(to: CGPoint(x: diamondEnd.x, y: diamondEnd.y + diamondSize / 2))
+                            path.addLine(to: CGPoint(x: diamondEnd.x - diamondSize / 2, y: diamondEnd.y))
+                            path.closeSubpath()
+                        }
+                        .fill(Color.purple)
+                    }
+                    Button(action: {
+                        guard let tempRange = scheduleTemporarySelectedGridRange else { return }
+                        let startRow = tempRange.start.row
+                        let startCol = tempRange.start.col + viewStore.shiftedCol
+                        let endCol = tempRange.end.col + viewStore.shiftedCol
+                        let newLine = DrawnLine(startRow: startRow, startCol: startCol, endRow: startRow, endCol: endCol)
+                        self.completedLines.append(newLine)
+                        viewStore.send(.addCompletedLine(newLine))
+                        scheduleTemporarySelectedGridRange = nil
+                        self.drawingLine = nil
+                    }) {
+                        EmptyView()
+                    }
+                    .keyboardShortcut(.defaultAction)
+                    .hidden()
```

Comment:
```suggestion
       Button  {
                        guard let tempRange = scheduleTemporarySelectedGridRange else { return }
                        let startRow = tempRange.start.row
                        let startCol = tempRange.start.col + viewStore.shiftedCol
                        let endCol = tempRange.end.col + viewStore.shiftedCol
                        let newLine = DrawnLine(startRow: startRow, startCol: startCol, endRow: startRow, endCol: endCol)
                        self.completedLines.append(newLine)
                        viewStore.send(.addCompletedLine(newLine))
                        scheduleTemporarySelectedGridRange = nil
                        self.drawingLine = nil
                    } label: {
                        EmptyView()
                    }
                    .keyboardShortcut(.defaultAction)
                    .hidden()
```
위처럼 괄호를 최소화해서 가독성을 높일 수 있도록 작성하면 좋대요!

Reply:
action 없이 최소화하는 부분 통일하기로 했었는데 또 넣어버렸네요.. 지적해줘서 고마어요 가온 👍